Team54/src/main/resources/data-postgres.sql

```diff
@@ -1 +1,317 @@
+
+insert into vacation_request values (1, 0,'2021-09-11', '2021-08-21', 1, 9);
+insert into vacation_request values (2, 0,'2021-10-15', '2021-09-26', 2, 10);
+insert into vacation_request values (3, 0,'2021-03-22', '2021-02-19', 3, 17);
+
+insert into promotion values (1,'2021-03-22', '2021-02-19', '20 posto popusta', 1);
+insert into promotion values (2,'2021-05-01', '2021-04-20','10 posto popusta' , 1);
+insert into promotion values (3,'2021-11-20', '2021-11-26', '20 posto popusta', 2);
+insert into promotion values (4,'2021-02-22', '2021-02-17', '30 posto popusta', 2);
+
+
+insert into drug_in_storage values ( 1, 20, 1, 15);
+insert into drug_in_storage values ( 2, 24, 2, 15);
+insert into drug_in_storage values ( 3, 50, 3, 15);
+insert into drug_in_storage values ( 4, 40, 4, 16);
+insert into drug_in_storage values ( 5, 65, 5, 16);
+insert into drug_in_storage values ( 6, 25, 1, 16);
+insert into drug_in_storage values ( 7, 35, 2, 19);
+insert into drug_in_storage values ( 8, 15, 3, 19);
+insert into drug_in_storage values ( 9, 10, 4, 19);
+
+insert into supplier_drugs_in_storage values ( 15, 1);
+insert into supplier_drugs_in_storage values ( 15, 2);
+insert into supplier_drugs_in_storage values ( 15, 3);
+insert into supplier_drugs_in_storage values ( 16, 4);
+insert into supplier_drugs_in_storage values ( 16, 5);
+insert into supplier_drugs_in_storage values ( 16, 6);
+insert into supplier_drugs_in_storage values ( 19, 7);
+insert into supplier_drugs_in_storage values ( 19, 8);
+insert into supplier_drugs_in_storage values ( 19, 9);
+
+insert into orders values ( 1, '2021-02-17', 0, 13);
+insert into orders values ( 2, '2021-06-22', 0, 13);
+insert into orders values ( 3, '2021-05-08', 0, 14);
+insert into orders values ( 4, '2021-01-22', 0, 14);
+insert into orders values ( 5, '2021-12-11', 0, 14);
+
+	/*-- DERMATOLOGISTS IN PHARMACY --*/
+
+insert into dermatologists_in_pharmacy(pharmacy_id,dermatologist_id) values (1,1);
+insert into dermatologists_in_pharmacy(pharmacy_id,dermatologist_id) values (1,2);
+insert into dermatologists_in_pharmacy(pharmacy_id,dermatologist_id) values (1,3);
+insert into dermatologists_in_pharmacy(pharmacy_id,dermatologist_id) values (1,4);
+insert into dermatologists_in_pharmacy(pharmacy_id,dermatologist_id) values (2,1);
+insert into dermatologists_in_pharmacy(pharmacy_id,dermatologist_id) values (2,3);
+
+	/*-- SCHEDULED EXAMINATIONS -DERMATOLOGIST  --*/
+
+insert into examination(diagnose,emplyeed_id,price,status,duration,start,therapy_duration,type,patient_id,pharmacy_id)
+ values('Ekcem',1,20,0,30,'2021-06-01 12:00',30,0,5,1);
+ insert into examination(diagnose,emplyeed_id,price,status,duration,start,therapy_duration,type,patient_id,pharmacy_id)
+ values('Alergija',1,20,0,30,'2021-06-07 12:00',30,0,5,1);
+ insert into examination(diagnose,emplyeed_id,price,status,duration,start,therapy_duration,type,patient_id,pharmacy_id)
+ values('Suga',1,20,0,30,'2021-06-15 12:00',30,0,5,1);
```

Comment:
Za sve preglede, treba dodati nekolicinu vec zavrsenih pregleda(i status i type polje  u modelu treba dodati anotaciju enumtype kao u "LoyaltyExaminationPoints" klasi) i nekolicinu pregleda koja ce se tek odrzati, samim tim nece imati dijagnozu niti trajanje terapije. Takodje, paziti sta se pise za dijagnozu jer sirup za kasalj nije dijagnoza.

Reply:
U klasi "" LoyaltyExamiantionPoints" se ne nalazi konkretan primer predlozenog resenja. Zar trajanje nece biti unapred odredjeno?

Reply:
Mislio da pogledas tu klasu za ova enum polja. Na ovoj je grani, nije na developu. Ja cu dodati to.

Reply:
Dodala sam ja to